src/components/PopupWithForm.js

```diff
@@ -2,32 +2,44 @@ import Popup from "./Popup.js";
 
 export default class PopupWithForm extends Popup {
   constructor(popupSelector, handleFormSubmit) {
-    super({popupSelector});
+    super({ popupSelector });
     this._handleFormSubmit = handleFormSubmit;
-    this._form = this._popupElement.querySelector('.modal__form');
+    this._form = this._popupElement.querySelector(".modal__form");
   }
 
   _getInputValues() {
     const inputValues = {};
     this._inputList = this._form.querySelectorAll(".modal__input");
```

Comment:
**Could be improved**

All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` , `this._likeButton` and so on.